assets/quick-add.css

```diff
@@ -77,6 +77,21 @@
   max-width: 100%;
 }
 
+.quick-add-modal__content-info--bulk h3 {
+  margin-bottom: 0.5rem;
+  margin-top: 0;
+}
+
+.quick-add-modal__content-info--bulk .price, .quick-add-modal__content-info--bulk .card__information-volume-pricing-note {
+  display: inline-block;
+}
+
+@media screen and (min-width: 750px) {
+  .quick-add-modal__content-info--bulk .card__information-volume-pricing-note {
+    padding-left: 1rem;
```

Comment:
```suggestion
    padding-left: 1.6rem;
```
How do you feel with this spacing? To help distinguish the price from the additional information?
![](https://screenshot.click/13-46-0str3-4e2iw.png)